### PR TITLE
update expansion to use the latest papi version

### PR DESCRIPTION
**Pull request by alaajel**

### CODE REVIEW

**Comment by alaajel** [comment]
Is this pr is being ignored?

**Comment by alaajel** [comment]
Bumping this again.....

**Comment by GeleVla** [comment]
Would be nice if it gets updated!

File: pom.xml [hunk 1]
```diff
@@ -64,21 +64,21 @@
...
             <artifactId>spigot-api</artifactId>
-            <version>1.15.2-R0.1-SNAPSHOT</version>
+            <version>1.16.5-R0.1-SNAPSHOT</version>
             <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>me.clip</groupId>
             <artifactId>placeholderapi</artifactId>
-            <version>2.9.1</version>
+            <version>2.10.1</version>
```
Comment (darbyjack) [inline pom.xml]: Latest is 2.10.9, might want to compile against that just to be sure everything still works.

File: src/main/java/com/helpchat/redisbungeeexpansion/RedisBungeeExpansion.java [hunk 2]
```diff
@@ -55,8 +54,23 @@ public boolean register() {
                 servers.put(s, 0);
             }
         }
+        boolean registered;
+        try {
+            //for older version of papi support.
```
Comment (darbyjack) [inline src/main/java/com/helpchat/redisbungeeexpansion/RedisBungeeExpansion.java]: I'd honestly say drop support for older versions of PAPI. They don't receive support from us anymore. Compile against latest, ensure it works on latest, don't worry about doing hacks to make it work on previous versions.

**Review by darbyjack — CHANGES_REQUESTED** [review]

**Comment by alaajel** [comment]
alright all requested changes will be pushed now.

**Comment by darbyjack** [comment]
LGTM. Do you have any screenshots showing it's working? I don't have a RedisBungee environment setup.

**Comment by alaajel** [comment]
image
image

**Review by darbyjack — APPROVED** [review]

**Pull request merged**